en/setup/dev_env_windows_cygwin.md

```diff
@@ -1,202 +1,225 @@
 
 Legacy Versions (**deprecated**):
 
 * [PX4 Windows Cygwin Toolchain 0.1 Download](https://s3-us-west-2.amazonaws.com/px4-tools/PX4+Windows+Cygwin+Toolchain/PX4+Windows+Cygwin+Toolchain+0.1.msi) (23.02.2018)
+-->
 
-## Support / Known Problems
+## Features / Issues
 
-Tested and working:
+The following features are known to work (version 2.0):
 
-* Building and running SITL with jMAVsim, generates a native windows binary px4.exe and has a lot better prformance than a VM
-* Building and uploading NuttX builds like px4fmu-v2 and px4fmu-v4
-* Style check with astyle with the command make format
+* Building and running SITL with jMAVSim with significantly better performance than a VM (it generates a native windows binary **px4.exe**).
+* Building and uploading NuttX builds (e.g.: px4fmu-v2 and px4fmu-v4)
+* Style check with *astyle* (supports the command: `make format`)
 * Command line auto completion
-* The installer does **NOT** screw up your system and global path, it only modifies the selected installation directory e.g. `C:\PX4\` and uses a temporary local path
+* Non-invasive installer! The installer does NOT affect your system and global path (it only modifies the selected installation directory e.g. **C:\PX4\** and uses a temporary local path).
 * The installer supports updating to a new version keeping your personal changes inside the toolchain folder
 
-[Known problems / Report your issue](https://github.com/orgs/PX4/projects/6)
+Omissions:
+* Simulation: Gazebo and ROS are not supported
+* Only NuttX and JMAVSim/SITL builds are supported.
+* [Known problems / Report your issue](https://github.com/orgs/PX4/projects/6)
 
 ## Installation Instructions
 
-1. Download the latest version of the ready to use MSI installer above
+1. Download the latest version of the ready to use MSI installer: [PX4 Windows Cygwin Toolchain 0.2](https://s3-us-west-2.amazonaws.com/px4-tools/PX4+Windows+Cygwin+Toolchain/PX4+Windows+Cygwin+Toolchain+0.2.msi) (09.05.2018)
 1. Run it, choose your desired installation location, let it install
     ![jMAVSimOnWindows](../../assets/toolchain/cygwin_toolchain_installer.PNG)
-1. Make sure to tick the box in the end of the installation to clone the PX4 repository, build and run simulation with jMAVsim this simplifies the process to get you started. 
-
-> **Note** Don't worry if you missed the opportunity or the script failed (e.g. because you didn't have a working internet connection). You can also [do that step manually on the console afterwards](#getting_started).
-
+1. Tick the box at the end of the installation to *clone the PX4 repository, build and run simulation with jMAVSim* (this simplifies the process to get you started). 
+
+   > **Note** If you missed this step will need to [clone the PX4 Firmware repository manually](#getting_started).
+
+
+## Getting Started {#getting_started}
```

Comment:
@MaEtUgR  OK, so IMO getting started should always be one of the first things you run into. Also I found the fact that this talked about the installer confusing "structurally". So now this explains what you do, but notes that some bits are optional.